themes/default/content/registry/packages/eks/how-to-guides/access-entries-migration.md

```diff
@@ -0,0 +1,453 @@
+    {
+        { "myRole", new Eks.Inputs.AccessEntryArgs
+        {
+            PrincipalArn = "arn:aws:iam::123456789012:role/eks-cluster-role",
+            Username = "test-role",
+            KubernetesGroups = new()
+            {
+                "test-group",
+            },
+            Type = Eks.AccessEntryType.Standard,
+        } },
+        { "myUser", new Eks.Inputs.AccessEntryArgs
+        {
+            PrincipalArn = "arn:aws:iam::123456789012:user/test-user",
+            Username = "test-user",
+            KubernetesGroups = new()
+            {
+                "test-users",
+            },
+            Type = Eks.AccessEntryType.Standard,
+        } },
+    },
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language java %}}
+
+```java
+new Cluster("cluster", ClusterArgs.builder()
+    .authenticationMode(AuthenticationMode.ApiAndConfigMap)
+    .roleMappings(RoleMappingArgs.builder()
+        .roleArn("arn:aws:iam::123456789012:role/eks-cluster-role")
+        .username("test-role")
+        .groups("test-group")
+        .build())
+    .userMappings(UserMappingArgs.builder()
+        .userArn("arn:aws:iam::123456789012:user/test-user")
+        .username("test-user")
+        .groups("test-users")
+        .build())
+    .accessEntries(Map.ofEntries(
+        Map.entry("myRole", Map.ofEntries(
+            Map.entry("principalArn", "arn:aws:iam::123456789012:role/eks-cluster-role"),
+            Map.entry("username", "test-role"),
+            Map.entry("kubernetesGroups", "test-group"),
+            Map.entry("type", "STANDARD")
+        )),
+        Map.entry("myUser", Map.ofEntries(
+            Map.entry("principalArn", "arn:aws:iam::123456789012:user/test-user"),
+            Map.entry("username", "test-user"),
+            Map.entry("kubernetesGroups", "test-users"),
+            Map.entry("type", "STANDARD")
+        ))
+    ))
+    .build());
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+eksCluster:
+  type: eks:index:Cluster
+  properties:
+    authenticationMode: API_AND_CONFIG_MAP
+    roleMappings:
+      - roleArn: arn:aws:iam::123456789012:role/eks-cluster-role
+        username: test-role
+        groups:
+          - test-group
+    userMappings:
+      - userArn: arn:aws:iam::123456789012:user/test-user
+        username: test-user
+        groups:
+          - test-users
+    accessEntries:
+      myRole:
+        principalArn: arn:aws:iam::123456789012:role/eks-cluster-role
+        username: test-role
+        kubernetesGroups:
+          - test-group
+        type: STANDARD
+      myUser:
+        principalArn: arn:aws:iam::123456789012:user/test-user
+        username: test-user
+        kubernetesGroups:
+          - test-users
+        type: STANDARD
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+#### instanceRoles
+These map to Access Entries of type `EC2_LINUX`.
+
+The following shows how you'd add access entries for the instance roles:
+```ts
```

Comment:
Do we need to convert this one as well?

Reply:
Oh wow, totally missed this one in the wall of language choosers... Good catch!